### PR TITLE
🚧 feat: all transform

### PR DESCRIPTION
**Pull request by thecampagnards**
Toujours d'actu ce commit ?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-.idea
\ No newline at end of file
+.idea
+server/npm-debug.log
```
Comment (thecampagnards) [inline .gitignore]: server ?

File: client/package.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "docktor",
-  "version": "0.9.0",
+  "version": "1.0.0",
   "private": true,
-  "proxy": "http://localhost:8080/",
+  "proxy": "http://docktor:8080/",
```
Comment (thecampagnards) [inline client/package.json]: to rollback at the end

File: client/src/components/layout/ShellSocket.tsx [hunk 3]
```diff
@@ -24,7 +24,7 @@ export default class ShellSocket extends React.Component<IShellSocketProps> {
     if (loc.protocol === "https:") {
       uri = "wss:";
     }
-    uri += `//${loc.hostname}:`;
+    uri += `//docktor:`;
```
Comment (thecampagnards) [inline client/src/components/layout/ShellSocket.tsx]: to rollback at the end

File: .gitignore (hunk 1 above)
Comment (SmalkiGroove) [reply .gitignore]: auto generated, i removed it

File: server/handler/groups/transform.go [hunk 4]
```diff
@@ -0,0 +1,409 @@
...
+	daemon, err := db.Daemons().FindByIDBson(group.Daemon)
+	if err != nil {
+		log.WithFields(log.Fields{
+			"daemonID": group.Daemon,
+			"error":    err,
+		}).Error("Error when retrieving group daemon")
+		return c.JSON(http.StatusBadRequest, err.Error())
+	}
+
+	findService := func(services []types.Service, name string) types.Service {
```
Comment (thecampagnards) [inline server/handler/groups/transform.go]: function to Services or somewhere in types

File: server/types/transform.go [hunk 5]
```diff
@@ -33,103 +21,114 @@ func FindServiceName(init string, config types.ContainerJSON) string {
...
+	cdCmd := "cd /data"
+	mkdirCmd := fmt.Sprintf("mkdir %s", tmpServiceDir)
+	moveCmd := ""
 	for k, source := range sources {
-		cmd := []string{fmt.Sprintf("mv /data/%s /data/%s/%s", source, serviceName, targets[k])}
-		err = daemon.CmdContainer(sourceVolume, cmd)
-		if err != nil {
-			return
-		}
+		moveCmd += fmt.Sprintf("mv %s %s/%s ", source, tmpServiceDir, targets[k])
```
Comment (thecampagnards) [inline server/types/transform.go]: missing &&

File: client/package-lock.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (thecampagnards) [inline client/package-lock.json]: chmod 644

Comment (SmalkiGroove) [reply client/package-lock.json]: sinon on le fout dans le gitignore ?

**Pull request merged**